backend/serverless.yml

```diff
@@ -5,31 +5,36 @@
 # uses to deploy your code to your provider of choice
 
 # The `service` block is the name of the service
-service:
-  name: ${self:custom.appconfig.stackName}
+service: ${self:custom.appconfig.stackName}
 plugins:
-  - serverless-cloudside-plugin
   - serverless-offline
-
 package:
   individually: false
   excludeDevDependencies: true
-  include:
+  patterns:
+    - 'node_modules/**'
+    - '!node_modules/aws-sdk/**'
```

Comment:
The lambda function images already include the aws sdk - this is why it can be excluded here.